doc/loggable.md

```diff
@@ -197,6 +199,47 @@ class Article
 </doctrine-mapping>
 ```
 
+
+<a name="custom-logentry-class"></a>
+
+## Custom LogEntry class
+
+```php
+<?php
+
+namespace Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Loggable\Entity\MappedSuperclass\AbstractLogEntry;
+
+/**
+ *
```

Comment:
```suggestion
```
BTW, try to mimic the structure defined by the `@DoctrineAnnotation` CS ruleset (see #2507).